pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdTopics.java

```diff
@@ -632,14 +632,10 @@ void run() throws Exception {
                 "--force" }, description = "Close all producer/consumer/replicator and delete topic forcefully")
         private boolean force = false;
 
-        @Parameter(names = { "-d",
```

Comment:
Will this command fails if user keep "-d"  in command line?

Reply:
yes, this is a break change and we hope to release on next major branch( such as 1.11).
Detail in dev to [here](https://lists.apache.org/thread/wbny8x0b31jdkgoc9cjlc29gt9fdsm6v).

Reply:
> hide `-d` rather than remove for compatibility, then declare that this parameter is invalid and the new behavior.

Yes, I think it's better this way. Hide the parameter and print a warning log if user set this "-d false" explicitly.

Reply:
There is no such way of using `-d false` or `-d true`, the correct way is to specify `-d` or not to specify.